flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/validation/DefaultValidator.java

```diff
@@ -366,6 +368,18 @@ private Optional<String> validateResources(String component, Resource resource)
             return Optional.of(component + " resource memory parse error: " + iae.getMessage());
         }
 
+        String storage = resource.getEphemeralStorage();
```

Comment:
This will not catch misconfigured storage resources if memory is not defined.

Reply:
Got catch. Thanks!